### DISCUSSION TITLE
Can someone help me with understanding this method ..io.opentelemetry.api.trace.Span.storeInContext ?

### DISCUSSION THREAD

**Discussion by GokulShrikanth** [open-telemetry/opentelemetry-java-instrumentation#3111] [Q&A]
Hello, Is this method used to add a new span to an existing context?

**Comment by jkwatson** [comment]
Yes, this is an `ImplicitContextKeyed` interface method that enables usage like `Context.with(span)`.  You can use it directly as well, if you desire.
